src/Processing.js

```diff
@@ -3235,7 +3235,7 @@
         b = p.lerp(hsb1[2], hsb2[2], amt);
         rgb = p.color.toRGB(h, s, b);
         // ... and for Alpha-range
-        a = p.lerp(a1, a2, amt) * colorModeA;
+        a = p.lerp(a1, a2, amt) * colorModeA + 0.5 | 0;
```

Comment:
this needs parentheses. Right now it'll evaluate as `p.lerp(a1, a2, amt) * colorModeA + (0.5 | 0)`, as the bit operation is immediate, so actually turns the 0.5 into 0, leaving us without rounding.
```
a = (p.lerp(a1, a2, amt) * colorModeA + 0.5) | 0;
```